config/sipi.knora-test-config.lua

```diff
@@ -142,6 +142,11 @@ sipi = {
     loglevel = "DEBUG",
 }
 
+fileserver = {
+    docroot = './server',
+    docroute = '/server'
+}
```

Comment:
Yes, this is the new fileserver feature. I agree that the naming could be a bit clearer.